bookkeeper-server/src/test/java/org/apache/bookkeeper/client/LedgerMetadataTest.java

```diff
@@ -35,6 +38,36 @@
 
     private static final byte[] passwd = "testPasswd".getBytes(UTF_8);
 
+    @Test
+    public void testGetters() {
```

Comment:
there are tests (BookKeeperBuildersTest) in which I had to use a cast to LedgerHandle to access LedgerMetadata, maybe it would be good to update them as well

Reply:
@eolivelli I would prefer cleaning them up when we deprecate old api. otherwise, it might be unnecessarily I am going to touch a lot of places.

Reply:
Fine for me